src/main/java/frc/robot/RobotContainer.java

```diff
@@ -86,4 +92,15 @@ public void setAutonDefaultCommands() {
     driveBase.setBrakingMode(IdleMode.kBrake);
   }
 
+  public void configureButtonBindings() {
+    Trigger mainButtonTrigger = operatorJoystick.button(OperatorConstants.SHOOTER_MAIN_BUTTON_ID);
```

Comment:
Code is good -- I would maybe rename mainButtonTrigger to mainWheelTrigger and auxiliaryButtonTrigger to auxiliaryWheelTrigger.